### PR TITLE
feature (tag and tag-list): remove all occurrences of onCloseTag and adapt unit test

### PR DESCRIPTION
**Pull request by dantrau**
@plentymarkets/team-terra

#### Must be done by Terra

Testing

-   [ ] Person 1 (mandatory)
-   [x] Person 2 (mandatory)

---

#### Must be done by every developer

Please inform a member of Terra (@plentymarkets/team-terra) to get the upper part of the checklist done (with urgency or deadline).

Documentation

-   [x] Changelog (optional: relevant for every change which influences the API)
-   [x] [Google spreadsheet](https://docs.google.com/spreadsheets/d/1OINnux8TEoitV-qdAxqUQaf9oGI_fqFVwfRWenRFfhI/edit#gid=0) (relevant when deprecating public APIs or features)

Testing

-   [x] Unit Test (updated or created)

### CODE REVIEW

**Review by pweyrich — DISMISSED** [review]

**Comment by pweyrich** [comment]
Diese PRs gerne gleich gegen 6.X.X. Dann werden auch die checks durchlaufen.

**Review by pweyrich — APPROVED** [review]

**Review by pweyrich — CHANGES_REQUESTED** [review]
Changelog fehlt

**Review by pweyrich — DISMISSED** [review]

**Review by pweyrich — APPROVED** [review]

**Review by ChristianMoll — APPROVED** [review]

**Review by ChristianMoll — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 -   **terra-syntax-editor** has been removed. Use [ck-editor](https://ckeditor.com/ckeditor-4/) instead.
 -   **dependencies** removed peer dependency `ng2-ace-editor` since we no longer depend on it.
+-   **terra-tag** removed deprecated `onCloseTag` method.
```
Comment (pweyrich) [inline CHANGELOG.md:7]:
```suggestion
-   **terra-tag** removed deprecated `onCloseTag` output.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
...
+-   **terra-taglist** removed deprecated `onCloseTag` method.
```
Comment (pweyrich) [inline CHANGELOG.md:8]:
```suggestion
-   **terra-taglist** removed deprecated `onCloseTag` output.
```

**Review by pweyrich — APPROVED** [review]

**Review by ChristianMoll — APPROVED** [review]

**Pull request merged**